### PR TITLE
3 packages from igarnier/prbnmcn-dagger at 0.0.1

### PR DESCRIPTION
**Pull request by igarnier**
This pull-request concerns:
-`prbnmcn-dagger.0.0.1`: Probabilistic programming library
-`prbnmcn-dagger-gsl.0.0.1`: Probabilistic programming library: GSL-based samplers
-`prbnmcn-dagger-stats.0.0.1`: Probabilistic programming library: prbnmcn-stats-based samplers

---
* Homepage: http://github.com/igarnier/prbnmcn-dagger
* Source repo: git+https://github.com/igarnier/prbnmcn-dagger
* Bug tracker: http://github.com/igarnier/prbnmcn-dagger

---
## 0.0.1
- First release of `prbnmcn-dagger`, a library for probabilistic programming

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

**Comment by igarnier** [comment]
Missing test dependencies :facepalm: I'll fix that

**Comment by igarnier** [comment]
Well, it seems I'll have a hard time making the tests pass. The tests depend on the three packages and I don't want to have a fourth package dedicated to tests, so I just deactivated them (and I really don't like it). I'll welcome any suggestion.

**Comment by igarnier** [comment]
Closing, will reopen a fresh PR with an additional test package

File: packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam [hunk 1]
```diff
@@ -9,9 +9,13 @@ license: "MIT"
 homepage: "http://github.com/igarnier/prbnmcn-dagger"
 bug-reports: "http://github.com/igarnier/prbnmcn-dagger"
 depends: [
-  "dune" {>= "2.8" & >= "2.8"}
+  "dune" {>= "2.9"}
   "pringo" {>= "1.3"}
   "prbnmcn-cgrph" {= "0.0.2"}
+  "qcheck" {>= "0.17" & with-test}
+  "prbnmcn-dagger-gsl" {= "0.0.1" & with-test}
```
Comment (mseri) [inline packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam:16]: To make this kind of circular dependencies work, you should just add `post`:
```
"prbnmcn-dagger-gsl" {= "0.0.1" & with-test & post}
```

(same for the two lines below)

Comment (igarnier) [reply packages/prbnmcn-dagger/prbnmcn-dagger.0.0.1/opam:16]: Oh, good to know. Thanks for the feedback!

**Pull request closed without merging**